integrations/dd-sdk-android-okhttp/src/main/kotlin/com/datadog/android/okhttp/trace/TracingInterceptor.kt

```diff
@@ -649,7 +661,13 @@ internal constructor(
 
         // taken from W3CHttpCodec
         internal const val W3C_TRACEPARENT_KEY = "traceparent"
-        internal const val W3C_DROP_SAMPLING_DECISION = "00-%s-%s-00"
+        internal const val W3C_TRACESTATE_KEY = "tracestate"
+
+        // https://www.w3.org/TR/trace-context/#traceparent-header
+        internal const val W3C_TRACEPARENT_DROP_SAMPLING_DECISION = "00-%s-%s-00"
```

Comment:
these are very weird as they look similar with the `TRACE_PARENT_VALUE` declared in the class above but here the name says that is about sampling ?

Reply:
yes, because it is the same, but with sampling decision already included - which is a `drop`, meaning `00` in the end.

The class above it not public, it cannot be referenced/used from OkHttp package.